dev/core/src/com/google/gwt/dev/shell/log/ServletContextTreeLogger.java

```diff
@@ -22,6 +22,7 @@
 /**
  * Tree logger that logs servlet context information.
  */
```

Comment:
Do you think we need to add a doc for deprecation reason and the replacement if there is any?